annotationprocessor/build.gradle

```diff
@@ -19,15 +16,15 @@ def source_GIT_COMMIT = new File(buildDir, 'generated/source/GIT_COMMIT/main/jav
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre8:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
 
     implementation project(':annotations')
     implementation "io.realm:realm-annotations:$realm_version"
 
     kapt "com.google.auto.service:auto-service:1.0-rc2" // don't update yet, won't auto generate code
     compileOnly "com.google.auto.service:auto-service:1.0-rc2" // don't update yet, won't auto generate code
 
-    implementation 'com.squareup:javapoet:1.9.0'
+    implementation 'com.squareup:kotlinpoet:1.0.0-RC1'
```

Comment:
This doesn't break anything? Code is still auto generated?

Reply:
Sorry, I didn't mean for any one to review this yet. I'm working on another pr that is based off this branch. For now this is just for me to have the Kotlin poet library in the project.

Reply:
make javapoet to kotlinpoet change 
a seprate PR
then code gen code will need modifications